src/engraving/libmscore/mmrest.cpp

```diff
@@ -95,8 +96,9 @@ void MMRest::draw(mu::draw::Painter* painter) const
             x += symBbox(sym).width() + spacing;
         }
     } else {
-        QPen pen(painter->pen());
-        pen.setCapStyle(Qt::FlatCap);
+        using namespace mu::draw;
```

Comment:
I believe that in such simple cases, ` using namespace mu::draw;`  is not needed, or it should be added uniformly in all places, i.e., for example, at the beginning of each `draw` method. (rather than sticking in the middle of the method)